Added files via upload

lesleykhuu opened the pull request:
add/remove fields. prints tables correctly

lesleykhuu committed:
Added files via upload

add/remove fields. prints tables correctly

lesleykhuu committed:
Added files via upload

ontraportgithub commented on crmlogin/Contact.php:
```diff
@@ -1,160 +1,229 @@
+	@brief Gets the column from the database and returns it.
+	*/
+	public function getValue($value){
+		if(mysql_escape_string($value) == 'firstname'){
```
Why are you escaping it here. Mysql escape is usually when you want to put sometthing INTO the db.

lesleykhuu replied:
I don't use this function but I did remove the mysql escapes in my current version. I have not committed this change though

ontraportgithub commented on crmlogin/Contact.php:
```diff
@@ -1,160 +1,229 @@
+
+	public function getColFields($user){
+		require 'connection.php';
+		$sqll = "SELECT `field` FROM `fieldrelation` WHERE `user` = '$user'";
```
Don't trust user input, these need to be mysql escaped.

lesleykhuu replied:
I also did not use this method. Removed it

ontraportgithub commented on crmlogin/addField.php:
```diff
@@ -0,0 +1,121 @@
+		// echo $colNames[$i];
+	}
+
+	// $sql = "SELECT * FROM `contactlist` WHERE `user`='$user'";
```
Don't pepper code with comments, unless its used for some later testing, remember its in the repo so you can always look through the history to get.

lesleykhuu committed:
Added files via upload

lesleykhuu commented:
Still trying to clean up code. Created a new Table class in Table.php. Used in welcome.php, contactedit, removecontact to print table and addfield and removefield to print fields

lesleykhuu committed:
Added files via upload

Got rid of addfield, removefield, and initcontactcreate. Added editTable. Working on redirected from removefield2 to editTable with the nav tab on the removeField tab

lesleykhuu committed:
Added files via upload

just an update

lesleykhuu committed:
Added files via upload

lesleykhuu committed:
Added files via upload

lesleykhuu committed:
Add files via upload

added sort for welcome table

lesleykhuu committed:
most updated

actual contact edit should not work

lesleykhuu committed:
fix file

fix the clear in contactedit
